src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -63,7 +64,12 @@ const ReportActionItemSingle = (props) => {
 
     return (
         <View style={props.wrapperStyles}>
-            <Pressable style={styles.alignSelfStart} onPress={() => showUserDetails(props.action.actorEmail)}>
+            <Pressable
+                style={styles.alignSelfStart}
+                onPressIn={() => ControlSelection.block()}
+                onPressOut={() => ControlSelection.unblock()}
```

Comment:
@mollfpr This is sufficient right?
```suggestion
                onPressOut={ControlSelection.unblock}
```